contrib/scripts/kind.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+set -o pipefail
+
+# Sample usages
+# ./contrib/scripts/kind.sh
+# IPv6=1 ./contrib/scripts/kind.sh
+# NO_BUILD=1 ./contrib/scripts/kind.sh
+# NO_PROVISION=1 ./contrib/scripts/kind.sh
+
+# Enable IPv6 mode. By default using IPv4
+export 'IPv6'="${IPv6:0}"
+# Set NO_BUILD=1 to avoid building docker images again
+export 'NO_BUILD'="${NO_BUILD:0}"
+# Set NO_PROVISION=1 to use existing kind cluster
+export 'NO_PROVISION'="${NO_PROVISION:0}"
```

Comment:
I find it easier to deal with flags if they are in the "positive" tone. Meaning, we can have a `BUILD` flag that's by default `1` and the user would need to say `BUILD=0 ...` to _disable_ the building, instead of _enabling_ the "negative".

I know we have precedence for using the "negative" tone  in our Vagrantfiles, but maybe we can start to move away from that pattern. What do you think?